index.js

```diff
@@ -56,13 +57,20 @@ const init = async () => {
   // Then we load events, which will include our message and ready event.
   const evtFiles = await readdir("./events/");
   client.logger.log(`Loading a total of ${evtFiles.length} events.`);
-  evtFiles.forEach(file => {
+  await Promise.all(evtFiles.map(async (file) => {
```

Comment:
This change does nothing.

Reply:
This change makes the loader incredibly faster. Promise.all requires an array of promises.

Reply:
It actually does a lot, it loads 5x faster. Just test it yourself

Reply:
The old one does this:
Wait for file 1. 
Then Wait for file 2...
But the new one does this:
Start them at the same time, and wait for all to finish.

Reply:
my bad, I haven't ever thought of inserting something into the event loop in this manner